documentation/modules/adding-virt-provider.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_content-type: PROCEDURE
 [id="adding-virt-provider_{context}"]
-= Adding {a-virt} provider
+= Adding {a-virt} destination provider
 
-You can add {a-virt} provider to the {project-short} web console in addition to the default {virt} provider, which is the provider where you installed {project-short}.
+You can add {a-virt} destination provider to the {project-short} web console in addition to the default {virt} provider, which is the provider where you installed {project-short}.
```

Comment:
@istein1 -- please this check this sentence carefully -- I added "destination" before only instance of "provider" in this sentence.

Reply:
Please add the bold 'destination'
You can add {a-virt} destination provider to the {project-short} web console in addition to the default {virt} **destination** provider, which is the provider where you installed {project-short}.


The idea here is that MTV VMs migration can be either to the default destination Openshift cluster named 'host',
or a user can migrate the VMs to Another destination Openshift cluster

Reply:
Done.